docs/authoring.md

```diff
@@ -162,12 +159,33 @@ Consuming C#/WinRT components from MSIX-packaged applications is supported with
 
 C#/WinRT supports authoring out-of-process components that can be consumed by Windows Runtime compatible languages. Currently, consuming an out-of-process component is supported for managed C# apps with the use of a packaging project. This scenario requires creating a `WinRT.Host.runtimeconfig.json` file, as demonstrated in the [background task component sample](https://github.com/microsoft/CsWinRT/tree/master/src/Samples/BgTaskComponent). Native C++ consumption of out-of-process C# components is not fully supported yet.
 
+### Using Windows App SDK
+
+Authoring WinUI types with C#/WinRT is in development and is supported with some exceptions.
+Component authors can implement WinUI types using the Windows App SDK class library template. 
+Components are consumed by projects made from the various other Windows App SDK templates.

```

Comment:
How about "Components can be consumed by application projects that reference the Windows App SDK"?  

Someone could have an existing project (e.g. WPF) that references the Windows App SDK, not necessarily created from the templates